src/util/metrics.ts

```diff
@@ -0,0 +1,64 @@
+import express from 'express';
+import { Application, Request, Response } from 'express';
+import client from 'prom-client';
+
+import { Log } from '../logging/Log';
+import { parseArgs } from '../parseArgs';
+
+const { logger } = Log;
+
+const register = new client.Registry();
+
+export const httpErrorCounter = new client.Counter({
+	name: 'sas_http_errors',
+	help: 'Number of HTTP Errors',
+});
+
+register.registerMetric(httpErrorCounter);
```

Comment:
Same with these two calls.

Reply:
I moved the `register...` under `metricsEndpoint`. Do I have to move also the `httpErrorCounter` ? Since this is a `const` that is exported so I should not move it right?